recipes/icu/icu-config.patch

```diff
@@ -0,0 +1,11 @@
+--- source/config/Makefile.inc.in.orig	2012-11-26 11:59:38 +0400
++++ source/config/Makefile.inc.in	2012-11-26 12:08:56 +0400
+@@ -41,7 +41,7 @@
+ # The prefix for ICU libraries, normally 'icu'
+ ICUPREFIX = icu
+ PACKAGE = @PACKAGE@
+-LIBICU = lib$(ICUPREFIX)
++LIBICU = $(ICUPREFIX)
```

Comment:
On Windows, DLLs usually don't have lib prefixes.  This is neither here nor there.  My experience with this library was that whatever I was working on (I forgot) that needed it needed it without the lib prefix.

This is common, I find - many packages go looking for libraries with names other than the standard ones we build.  How should we approach this?  I see a few options:

  - copy/link file in recipe providing file
  - copy/link file in recipe using file
  - if only one thing is using a library, rename output, as done here.